lib/IDE/CodeCompletion.cpp

```diff
@@ -5769,6 +5866,16 @@ void swift::ide::copyCodeCompletionResults(CodeCompletionResultSink &targetSink,
 
       llvm_unreachable("Unhandled CodeCompletionDeclKind in switch.");
     });
+  } else if (onlyPrecedenceGroups) {
+    std::copy_if(sourceSink.Results.begin(), sourceSink.Results.end(),
+                 std::back_inserter(targetSink.Results),
+                 [](CodeCompletionResult *R) -> bool {
+      if (R->getAssociatedDeclKind() ==
+          CodeCompletionDeclKind::PrecedenceGroup) {
+        return true;
+      }
+      return false;
```

Comment:
This could be simplified to `return R->getAssociatedDeclKind() == CodeCompletionDeclKind::PrecedenceGroup`.